docs/.vuepress/config.js

```diff
@@ -234,19 +244,36 @@ module.exports = {
                   path: "/apps/interchain-accounts/parameters.html",
                 },
                 {
-                  title: "Development Use Cases",
+                  title: "Client",
                   directory: false,
-                  path: "/apps/interchain-accounts/development.html",
+                  path: "/apps/interchain-accounts/client.html",
                 },
                 {
                   title: "Active Channels",
                   directory: false,
                   path: "/apps/interchain-accounts/active-channels.html",
                 },
                 {
-                  title: "Authentication Modules",
-                  directory: false,
-                  path: "/apps/interchain-accounts/auth-modules.html",
+                  title: "Legacy",
```

Comment:
I added like a subfolder with the legacy related documentation, so that when we are ready to eliminate the legacy APIs, we can just remove these files at that should be it (from a documentation updating point of view).